src/main/java/com/benko/exercise/sort/BubbleSorter.java

```diff
@@ -0,0 +1,34 @@
+package com.benko.exercise.sort;
+
+public class BubbleSorter {
+
+    public void sort(int[] numbers) {
+        for (int i = 0; i < numbers.length; i++) {
+            swapElements(numbers);
+        }
+        printNumbers(numbers);
+    }
+
+    private static int[] swapElements(int[] numbers) {
+        for (int i = 0; i < numbers.length-1; i++) {
+            if (isGeraterThan(i, i+1, numbers)) {
+                int temp;
+                temp = numbers[i + 1];
+                numbers[i + 1] = numbers[i];
+                numbers[i] = temp;
+            }
+        }
+        int[] swapedElements = numbers;
+        return swapedElements;
```

Comment:
There's no reason for this method to return anything. Method should be void.
Besides:
* you ain't using the returned value,
* it doesn't look good to init new variable just before returning it. For the future, remember that instead of
```
        int[] swapedElements = numbers;
        return swapedElements;
```
it's better to do just:
```
return numbers;
```

Reply:
You're right! I copied your resolution from somewhere but without understanding, now I got it!
Done